pkg/cmd/search/repos/repos.go

```diff
@@ -102,7 +102,7 @@ func NewCmdRepos(f *cmdutil.Factory, runF func(*ReposOptions) error) *cobra.Comm
 	cmdutil.StringEnumFlag(cmd, &sort, "sort", "", "best-match", []string{"forks", "help-wanted-issues", "stars", "updated"}, "Sort fetched repositories")
 
 	// Query qualifier flags
-	cmdutil.NilBoolFlag(cmd, &opts.Query.Qualifiers.Archived, "archived", "", "Filter based on archive state")
+	cmdutil.NilBoolFlag(cmd, &opts.Query.Qualifiers.Archived, "archived", "", "Filter based on archive state, '--archived=false' to exclude archived repositories")
```

Comment:
@shion1305 @samcoe: would creating a more elaborate section in the `Long` usage help docs make more sense, explaining both `--archived=true` and `--archived=false`?